cloudconfig/cloudinit/cloudinit_opensuse.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2015 Canonical Ltd.
+// Copyright 2015 Cloudbase Solutions SRL
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package cloudinit
+
+import (
+	"github.com/juju/utils/packaging/commands"
+	"github.com/juju/utils/proxy"
+)
+
+//Implementation of PackageHelper for OpenSUSE
+type openSUSEHelper struct {
+	paccmder commands.PackageCommander
+}
+
+//Returns the list of required packages in OpenSUSE
+func (helper openSUSEHelper) getRequiredPackages() []string {
+	return []string{
+		"curl",
+		"bridge-utils",
+		//"cloud-utils", Put as a requirement to the cloud image (requires subscription)
```

Comment:
No, cloud-init package is in the public repo. Other packages (like cloud-utils) should be in the cloud repo.

For public clouds (I checked aws), you can deploy openSUSE with access to that repo (the prices of this option is ten times greater than deploying ubuntu, so I guess that you are paying the fee/royalty in the instance cost)

For privates clouds I think that you need access to cloud repo. I will check this in my next step: to use OpenStack as provider.